lib/addons/flagger/my_new_addon.ts

```diff
@@ -0,0 +1,65 @@
+// filename=lib/kubevious_addon.ts
+import 'source-map-support/register';
+import * as blueprints from '../../../lib';
+import { Construct } from 'constructs';
+import { setPath } from '../../utils';
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {
+  version?: string,
+  ingressEnabled?: boolean,
+  flaggerServiceType?: string,
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+export const defaultProps: blueprints.HelmAddOnProps & FlaggerAddOnProps = {
+  name: "flagger-progressive-delivery", // Internal identifyer for our add-on
```

Comment:
this addon is `flagger`, let's call it that.

Reply:
Felt weird keeping name, namespace, and chart all "flagger" but I guess that is the proper naming scheme.

Reply:
There are cases when these can be different. For example if the same addon is added more than once to the blueprint. another case: containerinsights installes adot collector helm chart. However, in most cases they are identical.